tasks/ast_tools/src/generators/typescript.rs

```diff
@@ -52,11 +62,14 @@ fn generate_ts_type_def(type_def: &TypeDef, code: &mut String, schema: &Schema)
         // Empty string means don't output any TS def at all for this type
         if !custom_ts_def.is_empty() {
             write_it!(code, "export {custom_ts_def};\n\n");
+            unreachable!();
```

Comment:
If we're not using `#[estree(custom_ts_def)]` any more, it'd be preferable to remove it entirely (replace with just a `no_ts_def: bool` flag in `ESTreeStruct` and `ESTreeEnum`). Having `unreachable!()` here is a bit odd.

That change would ideally be in a separate PR, with this PR to add `Node` type stacked on top of it.

Reply:
Oops, I didn't mean to push this one. It was me quickly checking the code path :facepalm:
I'll split the PRs. Thanks for the suggestion :+1:

Reply:
Thank you!